suite-native/state/src/store.ts

```diff
@@ -48,6 +48,8 @@ export const initStore = async (preloadedState?: PreloadedState) =>
                 },
                 serializableCheck: false,
                 immutableCheck: false,
-            }).concat(middlewares),
+            })
+                .prepend(deviceConnectionMiddleware.middleware)
```

Comment:
Why prepend and not just adding it in the list of middlewares?

Reply:
I think I get it now, it is a listener middleware and needs to execute after actions have been processed.